lib/ide/parser/src/wsclient.rs

```diff
@@ -196,14 +197,15 @@ impl Client {
 }
 
 impl api::IsParser for Client {
-    fn parse(&mut self, program: String) -> api::Result<api::Ast> {
-        let request  = Request::ParseRequest { program };
+
+   fn parse(&mut self, program:String, ids:api::IDMap) -> api::Result<api::Ast> {
+        let request  = Request::ParseRequest { program, ids };
```

Comment:
We don't place spaces in inner sides of {…} braces.
We don't place space between simple arguments. (no one knows what exactly is "simple" but single identifier is simple)